mapbox/libandroid-telemetry/src/main/java/com/mapbox/services/android/telemetry/navigation/NavigationLocation.java

```diff
@@ -0,0 +1,58 @@
+package com.mapbox.services.android.telemetry.navigation;
+
+import android.location.Location;
+import android.util.JsonToken;
+
+import org.json.JSONArray;
+import org.json.JSONObject;
+
+import java.text.DateFormat;
+import java.text.SimpleDateFormat;
+import java.util.Date;
+import java.util.HashMap;
+import java.util.Locale;
+import java.util.TimeZone;
+
+public class NavigationLocation {
+
+  private static final String UTC_TIME_FORMAT = "yyyy-MM-dd'T'HH:mm:ss.SSSZ";
+  private static final String ACCURACY = "horizontalAccuracy";
```

Comment:
Almost all of these variables are duplicated.
👀 [`MapboxEvent`](https://github.com/mapbox/mapbox-java/blob/master/mapbox/libandroid-telemetry/src/main/java/com/mapbox/services/android/telemetry/MapboxEvent.java) and [`MapboxNavigationEvent`](https://github.com/mapbox/mapbox-java/blob/master/mapbox/libandroid-telemetry/src/main/java/com/mapbox/services/android/telemetry/navigation/MapboxNavigationEvent.java)

Reply:
doesn't matter, this is temporary